### DISCUSSION TITLE
How to limit allowed HTML tags for react-remark

### DISCUSSION THREAD

**Discussion by meotimdihia** [remarkjs/remark#763] [Q&A] [answered]
How to limit allowed tags for [react-remark.](https://github.com/remarkjs/react-remark) This is my currently code: 
```javascript
    <Remark
      remarkToRehypeOptions={{ allowDangerousHtml: true }}
      rehypePlugins={[rehypeRaw, rehypeSanitize]}
    >
      {content}
    </Remark>
```
I want to use react-remark to render comments were created by users. Please help me to update the above code to allow only HTML tags: `a`, `img`, `b`, `strong`, `em`, `detail`

**Comment by ChristianMurphy** [comment, marked as answer]
```tsx
<Remark
  remarkToRehypeOptions={{ allowDangerousHtml: true }}
  rehypePlugins={[rehypeRaw, [rehypeSanitize, {tagNames: ["a", "img", "b", "strong", "em", "detail"]}]]}
>
  {content}
</Remark>
```

for additional details on other schema checking possible, see https://github.com/syntax-tree/hast-util-sanitize#schema
